Gemfile.lock

```diff
@@ -51,8 +51,8 @@ GEM
       rack (>= 1.0.0)
       rack-test (>= 0.5.4)
       xpath (~> 2.0)
-    capybara-webkit (1.3.1)
-      capybara (>= 2.0.2, < 2.5.0)
+    capybara-webkit (1.5.1)
```

Comment:
Hmm...did you do a bundle update?

This is fine, but normally I wouldn't expect to see Gemfile.lock changes.